### PR TITLE
Map Components Polish 2: Add top shadow

### PR DESCRIPTION
**Pull request by yvonnesjy**
This adds a subtle shadow on top of the map to soften the transition from the top navigation bar to the map.

Before
image

After
image

### CODE REVIEW

**Comment by robyngit** [comment]
Tested this but then realized you've only set Ian as the reviewer, oopsie! Here is what I found:

- The issue in https://github.com/NCEAS/metacatui/pull/2353 is included in these changes.

- The help panel screenshot includes this new feedback blurb, which isn't actually included in these changes: Screenshot 2024-04-11 at 10 55 34 AM
  (BTW, where will we source the contact email from when we do decide to implement this feedback blurb?)

**Comment by yvonnesjy** [comment]
> The issue in https://github.com/NCEAS/metacatui/pull/2353 is included in these changes.

Hey yup, this branch is based on the 2353 branch (and all my styling changes are kinda based on each other now). In general, I name my _initial_ commit for a new PR `<issue name>: <change summary>` and the followup commits `Fixup: <change summary>`. So when you're reviewing, you can select the _last_ commit that _doesn't_ start with `Fixup` and all the fixups that follow it.

> The help panel screenshot includes this new feedback blurb

That was from https://github.com/NCEAS/metacatui/pull/2348/files and the screenshot **with** the feedback blurb actually existed in the repo, and was replaced by one **without** the blurb. I just thought it was an outdated screenshot cause I haven't seen blurb anywhere... but lmk if that's not the case😳

**Review by ianguerin — APPROVED** [review]

**Review by robyngit — CHANGES_REQUESTED** [review]
Reminder:
[ ] Ensure that the expanded layer panel doesn't overlap with the map in the Data Catalog View.

**Comment by yvonnesjy** [comment]
> Ensure that the expanded layer panel doesn't overlap with the map in the Data Catalog View. 

Rebased and this should be fixed now.

File: src/css/map-view.css [hunk 1]
```diff
@@ -339,7 +339,7 @@
   max-height: 100%;
   overflow: auto;
   /* required to be placed above map widget in firefox: */
-  z-index: 1;
+  z-index: 3;
```
Comment (ianguerin) [inline src/css/map-view.css:342]: In my experience this is the kind of thing that is destined to be a regression (especially without screenshot tests). One small thing that has worked for me is defining a variable for the z-index you're interested in, and then doing some math based on that z-index. So if you have 
```
--map-widget-z-index: 1; 
```

Then here you could have
```
z-index: calc(2 + var(--map-widget-z-index);
```

**Review by ianguerin — APPROVED** [review]

File: src/css/map-view.css (hunk 1 above)
Comment (robyngit) [reply src/css/map-view.css:342]: ^ good tip @ianguerin, I'm going to use that

Comment (yvonnesjy) [reply src/css/map-view.css]: much better!

**Pull request merged**